Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -101,29 +102,32 @@ sub _releases {
 			$_->{release_type} = 'COMPILATION';
 			$addToMainReleases->();
 			# only list outside the compilations if Composer/Conductor
-			next unless $_->{role_ids} =~ /COMPOSER|CONDUCTOR/ && $_->{role_ids} !~ /ARTIST|BAND/;
+			next unless ( $defaultRoles =~ /COMPOSER|CONDUCTOR/ || $userDefinedRoles ) && $defaultRoles !~ /ARTIST|BAND/;
```

Comment:
I think we should use stricter matching. `/\b(ARTIST|BAND)\b/` would check for word boundaries (not matching `TRACKARTIST`). Unless the latter was expected, of course. With the arrival of user defined roles we might hit the unexpected.

Reply:
It's only checking `$defaultRoles`. This line of code has always confused me a bit, so I'm not sure whether it should be matching `TRACKARTIST` or not, though it is at present.

Reply:
Ok, let's not touch this for now.